frog/imports/client/Wiki/index.js

```diff
@@ -213,7 +224,7 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
     const instanceId =
       this.getInstanceIdForName(
         fullPageObj,
-        this.props.match.params.instance
+        this.props.match?.params.instance || this.props.embeddedPage?.instance
```

Comment:
this.props.embeddedPage.instance does not exist in embeddedPage

Reply:
Reverted to using `instance` instead of `instanceId`